qiskit/quantum_info/operators/symplectic/base_pauli.py

```diff
@@ -432,7 +436,7 @@ def _to_matrix(z, x, phase=0, group_phase=False, sparse=False):
             coeff = (-1j) ** phase
         else:
             coeff = 1
-        data = np.array([coeff * (-1) ** (bin(i).count("1") % 2) for i in z_indices & indptr])
+        data = coeff * _PARITY[(z_indices & indptr).astype(np.uint8)]
```

Comment:
Why is uint8 enough? Is it possible to have a larger number?

Reply:
The original code computes the parity of each value of `z_indices & indptr`.
My code first applies `astype(np.uint8)` to take mod 256 (parity does not change) and then applies a lookup table `_PARITY[256]` to compute the parity. Does it make sense?

Reply:
Thanks for your explanation. I misunderstood the definition of parity.

Reply:
Sorry. I noticed that I misunderstood the party of the code. This code does not work with many qubits. I will rethink of the optimization.

Reply:
fixed at fb1428c